config/jobs/etcd/etcd-periodics.yaml

```
@@ -174,3 +174,168 @@ periodics:
       # fuse needs privileged mode
       securityContext:
         privileged: true
+- name: ci-etcd-performance-ratio-1-128-amd64
+  interval: 24h
+  cluster: eks-prow-build-cluster
+  decorate: true
+  decoration_config:
+    timeout: 4h
+  extra_refs:
+    - org: etcd-io
+      repo: etcd
+      base_ref: main
+  annotations:
+    testgrid-dashboards: sig-etcd-periodics
+    testgris-tab-name: ci-etcd-performance-amd64
```

Typo on tab name? Need to fix on later jobs also.

```suggestion
    testgrid-tab-name: ci-etcd-performance-amd64
```